include/verbose.mk

```diff
@@ -30,12 +30,16 @@ ifeq ($(IS_TTY),1)
 endif
 
 define ERROR_MESSAGE
-  printf "$(_R)%s$(_N)\n" "$(1)" >&8
+  [ -L /proc/$$$$/fd/8 ] && \
```

Comment:
This is not portable and will break on non-Linux systems (e.g. macOS)

Reply:
does it have to be in the form:

`test [ expression ]`

in order to work on macs?

if I look at docs for POSIX and the `-L` operand is there....